src/main/java/com/xjeffrose/chicago/DefaultChicagoMessage.java

```diff
@@ -9,7 +9,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-@EqualsAndHashCode(exclude={})
+@EqualsAndHashCode(exclude={"decoderResult"})
```

Comment:
This was needed to allow for `.equals` to work properly on two different instances of DefaultChicagoMessage: one fed into encoder, one reconstructed by decoder.